src/services/goToDefinition.ts

```diff
@@ -230,14 +231,16 @@ export function getDefinitionAtPosition(program: Program, sourceFile: SourceFile
         const sigInfo = createDefinitionFromSignatureDeclaration(typeChecker, calledDeclaration, failedAliasResolution);
         // For a function, if this is the original function definition, return just sigInfo.
         // If this is the original constructor definition, parent is the class.
+        let includeDeclarations: ((d: Declaration) => boolean) | undefined;

```

Comment:
This is named like it's a boolean; I would suggest something like `declarationFilter`